README.md

```diff
@@ -25,4 +67,5 @@ Messages that do not conform to the schemas are still processed and stored. Howe
  
 
 ## Caveats
-- AGPS and PGPS do not have entries in the `deviceToCloud` directory because they both return a raw binary response (not JSON).
\ No newline at end of file
+- AGPS and PGPS do not have entries in the `deviceToCloud` directory because they both return a raw binary response (not JSON).
+- ts should be used instead of time. The time property is included to be backwards compatible with certain versions of asset tracker version 2 firmware. Future versions will use the ts property instead.
```

Comment:
Now that I see it here, I think it should also be placed above, right under the `time | ts` line, so it's very visible to people referring to this spec.

Reply:
How about we put a note "(See caveats)" so we're not duplicating and we are using the caveats section?

Reply:
Yes, sounds good